### PR TITLE
Dev/djo/docs full text search

### PR DESCRIPTION
**Pull request by nv-jbleau**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by nv-jbleau** [comment]
before: 
before
and after 
after

File: doc/templates/uno/styles/docfx.js [hunk 1]
```diff
@@ -515,6 +482,18 @@ $(function () {
         });
 
         renderSidebar();
+        if ($('#search-results').length !== 0) {
+          $('#search').show();
+          $('body').trigger("searchEvent");
+        }
+          // if the target of the click isn't the container nor a descendant of the container
```
Comment (nv-ksavaria) [inline doc/templates/uno/styles/docfx.js:489]: Indentation semble différente des lignes du dessus

File: doc/templates/uno/styles/docfx.js [hunk 2]
```diff
@@ -515,6 +482,18 @@ $(function () {
...
+          $('body').trigger("searchEvent");
+        }
+          // if the target of the click isn't the container nor a descendant of the container
+          $('body').mouseup(function (e) {
+              if (!$('#search-results').is(e.target) && $('#search-results').has(e.target).length === 0)
+              {
+                  $('#search-results').hide();
+              }
+          });
+
```
Comment (nv-ksavaria) [inline doc/templates/uno/styles/docfx.js:496]: Enlever whitespace

File: doc/templates/uno/styles/scss/main.scss [hunk 3]
```diff
@@ -68,11 +71,75 @@ h3 {
...
+  padding: 10px;
+
+  #search {
+    display: block;
+    border-radius: 10px;
+    input:focus {
+      outline: none;
+      border:2px solid #159bff;
+    }
+
```
Comment (nv-ksavaria) [inline doc/templates/uno/styles/scss/main.scss:85]: Enlever whitespace

File: doc/templates/uno/styles/scss/main.scss [hunk 4]
```diff
@@ -68,11 +71,75 @@ h3 {
...
+  }
+
+  #search-query {
+    border-radius: 6px;
+    height: 45px;
+    width: 100%;
+
```
Comment (nv-ksavaria) [inline doc/templates/uno/styles/scss/main.scss:92]: Enlever whitespace

File: doc/templates/uno/styles/scss/main.scss [hunk 5]
```diff
@@ -68,11 +71,75 @@ h3 {
...
+        color: black;
+        &:first-child {
+          border-radius: 10px 10px 0 0;
+        }
+        &:last-child {
+          border-radius: 0 0 10px 10px;
+          margin-bottom: 0;
+        }
+      }
+      p {
```
Comment (nv-ksavaria) [inline doc/templates/uno/styles/scss/main.scss:115]: Ajouter un linebreak avant le p

File: doc/templates/uno/toc.html.tmpl [hunk 6]
```diff
@@ -2,14 +2,14 @@
 
 <div id="sidetoggle">
   <div>
-    {{^_disableSideFilter}}
     <div class="sidefilter">
-      <form class="toc-filter">
-        <span class="glyphicon glyphicon-filter filter-icon"></span>
-        <input type="text" id="toc_filter_input" placeholder="Enter here to filter..." onkeypress="if(event.keyCode==13) {return false;}">
+      <form role="search" id="search" >
```
Comment (nv-ksavaria) [inline doc/templates/uno/toc.html.tmpl:6]: Revoir indentation

Comment (nv-ksavaria) [inline doc/templates/uno/toc.html.tmpl:6]: Enlever whitespace

**Review by nv-ksavaria — CHANGES_REQUESTED** [review]

**Review by davidjohnoliver — APPROVED** [review]

**Comment by davidjohnoliver** [comment]
> [CLA assistant check](https://cla-assistant.io/unoplatform/uno?pullRequest=5687)
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/unoplatform/uno?pullRequest=5687) before we can accept your contribution.
> **2** out of **3** committers have signed the CLA.
> 
> ✅ nv-jbleau
> ✅ davidjohnoliver
> ❌ nv-ksavaria
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/unoplatform/uno?pullRequest=5687) it.

@nv-ksavaria It's complaining that you haven't signed the license agreement, can you check?

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**